lib/config.js

```diff
@@ -101,7 +101,7 @@ export default class Config {
     // ensure server.watchOptions.ignored is an array (browsersync accepts string or array)
     // then push ['node_modules', outputDir] to make sure they're not watched
     res.server.watchOptions.ignored = Array.prototype.concat(res.server.watchOptions.ignored)
-    res.server.watchOptions.ignored = lodash.union(res.server.watchOptions.ignored, ['node_modules', res.outputDir])
+    res.server.watchOptions.ignored = union(res.server.watchOptions.ignored, ['node_modules', res.outputDir])
```

Comment:
I think you can do this with `Array.prototype.concat`

`Array.prototype.concat('foo', ['bar', 'baz']) // ['foo', 'bar', 'baz']